crates/core/src/node/vm.rs

```diff
@@ -1,18 +1,18 @@
 use zksync_multivm::{interface::storage::WriteStorage, vm_latest::Vm, HistoryMode};
 
-use super::boojumos::BoojumOsVM;
+use super::zksync_os::ZKsyncOSVM;
 
 #[allow(clippy::large_enum_variant)]
 pub enum AnvilVM<W: WriteStorage, H: HistoryMode> {
-    BoojumOs(BoojumOsVM<W, H>),
+    ZKsyncOS(ZKsyncOSVM<W, H>),
```

Comment:
Nit: rename variant to `ZksyncOs`. Bonus points for renaming `ZKSync` below to `Era`